harvdev_utils/examine_archives/search_archives.py

```diff
@@ -17,33 +17,40 @@
    # Later maybe Option to lookup specific value too.
 
    Usage examples
+   # Find examples of GA2c that have value and NOT blank
    python search_archives.py -l 5 -f GA2c
 
-   python search_archives.py -v Oseg3
+   # Find up to 999 examples maximum of Oseg1
+   python search_archives.py -v Oseg3 -l 999
 
+   # find examples of GA1a with the value 'zyd[JF01872]'
+   # NOTE: Safer to put ANY names with special chars like []!\ etc in quotes.
+   python search_archives.py -f GA1a -v 'zyd[JF01872]'
+
+   Disclaimer:
+      When specifying a field and a value it will NOT find examples where they are on seperate lines.
```

Comment:
oh, that's a _really_ good reminder - much harder to code for that scenario - glad you put that in there.